neurokit2/epochs/epochs_create.py

```diff
@@ -7,7 +7,8 @@
 from ..misc import listify
 
 
-def epochs_create(data, events, sampling_rate=1000, epochs_start=0, epochs_end=1, event_labels=None, event_conditions=None, baseline_correction=False):
+
+def epochs_create(data, events, sampling_rate=1000, epochs_start=0, epochs_end=1, phys_event=False, event_labels=None, event_conditions=None, baseline_correction=False):
```

Comment:
I am somehow not sure about this argument `phys_event` as it seems a bit too specific. Here's an alternative:

Currently, I think `epochs_start` and `epochs_end` accept lists, allowing for variable epochs lengths (although we should mention this possibility in the docstring). Thus, it's better if the user explicitly derives the end for each event (`end = np.insert(start[1:] - start[:-1], 0, start[0]); end = end/sampling_rate`) and then passes it to `epochs_end`. I think it's clearer and not really much more complicated. (and it forces an important step to be made explicit, which is a good thing IMO).

Reply:
You wrote : *I think `epochs_start` and `epochs_end` accept lists* - so I'll give it a try. Do some testing. 
But, I get what you are saying : functions have to stay simple and elegant.

Reply:
If we can do like you say without any problem, I'll just revert back to the last one commited in `neuropsychology/dev` and examplify in cardioresp notebook :)

Reply:
Amidst, all the back and forth with PRs, I forgot to mention that I had tried to derive variable length epochs by giving a list of the difference between successive onset. I had this error a couple of times :
`ValueError: The truth value of an array with more than one element is ambiguous. Use a.any() or a.all()`

I'll let you deliberate on the usefulness of my add-on, maybe let you try for yourself if you have time. In the mean time, I'll just remove my example from the PR.

Reply:
``site-packages\neurokit2\epochs\epochs_create.py in epochs_create(data, events, sampling_rate, epochs_start, epochs_end, phys_event, event_labels, event_conditions, baseline_correction)
``

```         #Find the maximum numbers in an epoch

--> 104         epoch_max_duration = int(max((i * sampling_rate for i in parameters["duration"])))
    105 
    106         # Then extend data by the max samples in epochs * NaN```